invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMemberBulkActions.js

```diff
@@ -0,0 +1,221 @@
+import { SearchResultsBulkActions } from "@js/invenio_communities/members/components/bulk_actions/SearchResultsBulkActions";
+import { ErrorMessage } from "@js/invenio_communities/members/components/ErrorMessage";
+import {
+  RolePermissionPolicy,
+  serializeOptionsByPermissions,
+  VisibilityPermissionPolicy,
+} from "../../components/bulk_actions/permissions";
+import { CommunityMembersApi } from "../../../api/members/api";
+import { BulkActionsContext } from "@js/invenio_communities/members/components/bulk_actions/context";
+import { RadioSelection } from "@js/invenio_communities/members/components/bulk_actions/RadioSelection";
+import { SelectedMembers } from "@js/invenio_communities/members/components/bulk_actions/SelectedMembers";
+import _mapValues from "lodash/mapValues";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_communities/i18next";
+import { Button, Form, Modal } from "semantic-ui-react";
+import { Trans } from "react-i18next";
+import { withState } from "react-searchkit";
+import { withCancel } from "react-invenio-forms";
+
+class ManagerMemberBulkActionsCmp extends Component {
```

Comment:
this component has a potential to improve it since it is handling too many responsibilities at once, but at the moment I didn't have a good idea on how to split it